### PR TITLE
[11.0][FIX] Copying dependencies in project_task_dependency

### PR DESCRIPTION
**Pull request by tarteo**
Fixes https://github.com/OCA/project/issues/422

### CODE REVIEW

File: project_task_dependency/models/project_task.py [hunk 1]
```diff
@@ -72,3 +72,16 @@ def _check_dependency_recursion(self):
             raise ValidationError(
                 _('You cannot create recursive dependencies between tasks.')
             )
+
+    @api.multi
+    def copy(self, default=None):
+        self.ensure_one()
+        if self.env.context.get('project_copy'):
+            res = super(ProjectTask, self).copy(default)
```
Comment (astirpe) [inline project_task_dependency/models/project_task.py]: This line could be moved before the `if`. This way you can avoid the `else`.

**Review by astirpe — CHANGES_REQUESTED** [review]

File: project_task_dependency/models/project_task.py (hunk 1 above)
Comment (astirpe) [reply project_task_dependency/models/project_task.py]: And this way the coverage will be not decreased also :)

**Review by elicoidal — APPROVED** [review]
LGTM

**Review by astirpe — APPROVED** [review]

File: project_task_dependency/__manifest__.py [hunk 2]
```diff
@@ -3,7 +3,7 @@
```
Comment (nikul-serpentcs) [inline project_task_dependency/__manifest__.py:3]: @tarteo IMHO Add website attribute, for reference  [here](https://github.com/OCA/maintainer-tools/blob/master/template/module/__manifest__.py#L10)

Comment (tarteo) [reply project_task_dependency/__manifest__.py:3]: Thanks, didn't notice it.

**Comment by tarteo** [comment]
Can this be merged? :)

**Review by nikul-serpentcs — APPROVED** [review]
Code Review LGTM

**Pull request merged**